SUMMARY.md

```diff
@@ -142,11 +142,3 @@
 	* [403 - Forbidden](messaging/errors/forbidden.md)
 	* [415 - Unsupported Content Type](messaging/errors/unsupportedContentType.md)
 	* [429 - Too Many Requests](messaging/errors/tooManyReq.md)
-
-## SDKs
```

Comment:
We'll need to keep these in the short term. Eventually they will link to the page that has a blurb about how to install the "single" SDK. Let's keep these pages and just add a place holder `csharp.md` , `java.md`, etc...

Reply:
Done!